library/core/src/primitive.rs

```diff
@@ -64,4 +64,6 @@ pub use u64;
 #[stable(feature = "core_primitive", since = "1.43.0")]
 pub use u8;
 #[stable(feature = "core_primitive", since = "1.43.0")]
+#[doc(alias = "size_t")]
```

Comment:
If we are adding these, then we should also add:
  - `_Bool` and `bool` for `bool`.
  - The exact-width integer types: `int8_t`, `uint8_t`, etc.
  - `char32_t` for `char`.
  - `ssize_t` for `isize` (POSIX, but common enough).
  - Perhaps `float` and `double` for `f32` and `f64` (not guaranteed, but likely).

Reply:
> _Bool and bool for bool.

I have the same question as in https://github.com/rust-lang/rust/pull/81989#discussion_r574660019 - what do you see as the point of `doc(alias)`? Both of those pull up `bool` as the first result in search already: https://doc.rust-lang.org/stable/std/?search=_Bool, https://doc.rust-lang.org/stable/std/?search=bool

The others look useful to add, thanks :)

Reply:
See my answer to that comment.

You're welcome! :)